oozie/src/main/java/org/apache/falcon/oozie/feed/FeedReplicationWorkflowBuilder.java

```diff
@@ -145,6 +161,24 @@ private String getDefaultMapBandwidth() {
         return RuntimeProperties.get().getProperty("falcon.replication.workflow.mapbandwidth", "100");
     }
 
+    private String getDefaultAmMemory() {
```

Comment:
These settings are not meant for other builders like ProcessWorfklowBuilder, might give wrong conscience of being available for other builders. Thats why it has been put into these two builders specifically. In case if we want to achieve the same for process builder too then we can move it into OrchestrationWorkflowBuilder.